install.sh

```diff
@@ -14,6 +11,13 @@ brew bundle
 brew unlink qt 2>/dev/null || true
 brew link --force qt5
 
+echo "Linking dotfiles into ~..."
+# Before `rcup` runs, there is no ~/.rcrc, so we must tell `rcup` where to look.
+# We need the rcrc because it tells `rcup` to ignore thousands of useless Vim
+# backup files that slow it down significantly.
+RCRC=rcrc rcup -v
```

Comment:
The `RCRC=rcrc` tells `rcup` to look for `./rcrc`, then using those settings it symlinks rcrc into place. This is bootstrapping the process of getting the rcrc in place.